model/manifest.go

```diff
@@ -119,13 +119,30 @@ type Manifest struct {
 }
 
 type ManifestBackend struct {
-	// The path to your executable binary. This should be relative to the root of your bundle and the
-	// location of the manifest file.
+	// Executables are the paths to your executable binaries, specifying multiple entry points
+	// for different platforms when bundled together in a single plugin.
+	Executables *ManifestExecutables `json:"executables,omitempty" yaml:"executables,omitempty"`
+
+	// Executable is the path to your executable binary. This should be relative to the root
+	// of your bundle and the location of the manifest file.
 	//
 	// On Windows, this file must have a ".exe" extension.
+	//
+	// If your plugin is compiled for multiple platforms, consider bundling them together
+	// and using the Executables field instead.
 	Executable string `json:"executable" yaml:"executable"`
 }
 
+type ManifestExecutables struct {
```

Comment:
Unmarshalling just into a `map[string]string` is painful in JSON, so I opted for the explicit approach here. Presumably if someone is compiling for another platform, this isn't onerous to maintain.